### ISSUE TITLE
NPE in Program.defects()

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#148]
```
java.lang.NullPointerException: Cannot invoke "java.lang.Iterable.iterator()" because the return value of "org.cactoos.scalar.IoChecked.value()" is null
    at org.eolang.lints.Program.defects (Program.java:80)
```

Similar to #130

**Comment by yegor256** [comment]
@volodya-lombrozo maybe you can help? I have no idea why this happens :((

**Comment by volodya-lombrozo** [comment]
@yegor256 Sure, I'll investigate it. Can you assign it to me then, please?

**Comment by yegor256** [comment]
@volodya-lombrozo assigned

**Comment by yegor256** [comment]
@volodya-lombrozo here, for example: https://github.com/objectionary/eo/actions/runs/12321135063/job/34391895857?pr=3632

**Comment by yegor256** [comment]
@volodya-lombrozo because of this bug we can't use lints 0.0.25 in EO

**Comment by volodya-lombrozo** [comment]
> @volodya-lombrozo because of this bug we can't use lints 0.0.25 in EO. Try to clone eo repo, upgrade lints to 0.0.25 and build: you will get the error. I suspect the problem is related to the switch of java classloaders that is done (maybe!) by the NLP library.

@yegor256 Unfortunately, It's impossible to build `eo` now. I've done what you suggested: cloned `eo`, upgraded `lints` to `0.0.25` and run `mvn clean package`. After it the entire process stacked ~ 30 min, after which I had to just cancel the build (`Ctrl-C`). 

Also I checked the logs, you sent, they don't say anything useful:

```
Caused by: java.lang.NullPointerException
    at org.eolang.lints.Program.defects (Program.java:80)
    at org.eolang.maven.LintMojo.lint (LintMojo.java:266)
    at org.eolang.maven.LintMojo.lambda$lintOne$1 (LintMojo.java:142)
    at org.cactoos.scalar.ScalarOf.lambda$new$1 (ScalarOf.java:71)
    at org.cactoos.scalar.ScalarEnvelope.value (ScalarEnvelope.java:53)
    at org.cactoos.text.TextOfScalar.asString (TextOfScalar.java:58)
    at org.cactoos.text.TextEnvelope.asString (TextEnvelope.java:51)
    at org.cactoos.bytes.BytesOf.lambda$new$2 (BytesOf.java:233)
    at org.cactoos.bytes.BytesOf.asBytes (BytesOf.java:391)
    at org.cactoos.io.InputOf.lambda$new$5 (InputOf.java:288)
    at org.cactoos.scalar.Checked.value (Checked.java:75)
    at org.cactoos.scalar.IoChecked.value (IoChecked.java:63)
    at org.cactoos.io.InputOf.lambda$new$6 (InputOf.java:289)
    at org.cactoos.io.InputOf.stream (InputOf.java:312)
    at org.cactoos.io.TeeInput.stream (TeeInput.java:867)
    at org.cactoos.scalar.LengthOf.lambda$new$2 (LengthOf.java:90)
    at org.cactoos.scalar.ScalarEnvelope.value (ScalarEnvelope.java:53)
    at org.cactoos.scalar.Checked.value (Checked.java:75)
    at org.cactoos.scalar.IoChecked.value (IoChecked.java:63)
    at org.eolang.maven.footprint.Saved.value (Saved.java:109)
    at org.eolang.maven.footprint.FpGenerated.apply (FpGenerated.java:51)
    at org.eolang.maven.footprint.FpFork.apply (FpFork.java:74)
    at org.eolang.maven.footprint.FpEnvelope.apply (FpEnvelope.java:50)
    at org.eolang.maven.footprint.FpFork.apply (FpFork.java:74)
    at org.eolang.maven.footprint.FpEnvelope.apply (FpEnvelope.java:50)
    at org.eolang.maven.footprint.FpEnvelope.apply (FpEnvelope.java:50)
    at org.eolang.maven.footprint.FpExistedSource.lambda$new$0 (FpExistedSource.java:44)
    at org.eolang.maven.footprint.FpEnvelope.apply (FpEnvelope.java:50)
    at org.eolang.maven.footprint.FpEnvelope.apply (FpEnvelope.java:50)
    at org.eolang.maven.LintMojo.lintOne (LintMojo.java:147)
    at org.eolang.maven.LintMojo.lambda$exec$0 (LintMojo.java:96)
    at org.eolang.maven.util.Threaded.lambda$total$0 (Threaded.java:75)
    at org.cactoos.scalar.CallableEnvelope.call (CallableEnvelope.java:53)
    at java.util.concurrent.FutureTask.run (FutureTask.java:264)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:829)
```

Ok, I will try to reproduce the problem somehow else.

**Comment by yegor256** [comment]
@volodya-lombrozo try `mvn clean install` (works on my machine, with lints 0.0.24, and works in CI)

**Comment by volodya-lombrozo** [comment]
@yegor256 It's the same command, but the `package` command is just one step before `install`. Most probably, you already have some caches which allows you to build faster. It's hard to say.

**Comment by yegor256** [comment]
@volodya-lombrozo just `package` won't work, since `eo-maven-plugin` needs `eo-parser`, which must be installed

**Comment by volodya-lombrozo** [comment]
@yegor256 ah sure, you are right, thank you. However, it doesn't help either, the build is still rather long. I guess, it's a separate issue though.

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#157](https://github.com/objectionary/lints/issues/157), [#158](https://github.com/objectionary/lints/issues/158) are still not solved.

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#157](https://github.com/objectionary/lints/issues/157), [#160](https://github.com/objectionary/lints/issues/160) are still not solved; solved: [#158](https://github.com/objectionary/lints/issues/158).

**Comment by 0pdd** [comment]
@yegor256 all 3 puzzles are solved here: [#157](https://github.com/objectionary/lints/issues/157), [#158](https://github.com/objectionary/lints/issues/158), [#160](https://github.com/objectionary/lints/issues/160).
